exercises/parallel-letter-frequency/.meta/src/reference/java/ParallelLetterFrequency.java

```diff
@@ -0,0 +1,45 @@
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+
+public class ParallelLetterFrequency {
+
+    private static Map<Integer, Integer> map;
```

Comment:
Maybe calling it something like `letterFrequencyMap` would make it more clear what it is? :)